Packs/CommonPlaybooks/Playbooks/playbook-Containment_Plan_-_Quarantine_File.yml

```diff
@@ -0,0 +1,427 @@
+description: |-
+  This playbook is one of the sub-playbooks in the containment plan.
+  This playbook handles the quarantining of files as a crucial step in the containment action.
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 0c4f76b8-840e-49d6-83ea-28853fed1128
+    type: start
+    task:
+      id: 0c4f76b8-840e-49d6-83ea-28853fed1128
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "7"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 70
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 397f4b57-3b4f-40ed-8871-59fb281d5a77
+    type: title
+    task:
+      id: 397f4b57-3b4f-40ed-8871-59fb281d5a77
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1365
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: a0ceb1ef-9c66-4295-8ab8-8ff9b8cbbf2f
+    type: regular
+    task:
+      id: a0ceb1ef-9c66-4295-8ab8-8ff9b8cbbf2f
+      version: -1
+      name: Get file quarantine status
+      description: Retrieves the quarantine status for a selected file.
+      script: '|||core-get-quarantine-status'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "13"
+    scriptarguments:
+      endpoint_id:
+        complex:
+          root: inputs.EndpointID
+      file_hash:
+        complex:
+          root: inputs.FileHash
+          filters:
+          - - operator: isNotEqualString
+              left:
+                value:
+                  simple: inputs.FileHash
+                iscontext: true
+              right:
+                value:
+                  simple: c:\windows\explorer.exe
+              ignorecase: true
+      file_path:
+        complex:
+          root: inputs.FilePath
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 210
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: 30398ddb-5182-4c19-84f2-6f06e62eabca
+    type: regular
+    task:
+      id: 30398ddb-5182-4c19-84f2-6f06e62eabca
+      version: -1
+      name: Auto file quarantine
```

Comment:
please use also the incident_id input with ${alert.id}

Reply:
Done